chart/kepler/templates/daemonset.yaml

```diff
@@ -68,35 +71,45 @@ spec:
           successThreshold: 1
           timeoutSeconds: 10
         volumeMounts:
-        - mountPath: /lib/modules
-          name: lib-modules
-        - mountPath: /sys
-          name: tracing
-        - mountPath: /proc
-          name: proc
-        - mountPath: /usr/src
-          name: usr-src
+          - name: lib-modules
+            mountPath: /lib/modules
+          - name: tracing
+            mountPath: /sys
+          - name: proc
+            mountPath: /proc
+          - name: usr-src
```

Comment:
would you please make this optional too? Turns GKE doesn't like it :D
https://github.com/sustainable-computing-io/kepler/issues/770#issuecomment-1631212965

Reply:
I am not sure what you mean that this. They have always been there

Reply:
@yellowhat you are right, kepler has been using `/usr/src` since start but in recent deployments on GKE, `/usr/src` is missing on the GKE instances are causing pod fail to start. If you can make it optional, that'll be great. Thanks

Reply:
Is it a different path or simply does not allow to mount?

Reply:
based on the error message, the path doesn't exist on GKE
```
  Warning  FailedMount  5m53s (x23 over 36m)  kubelet            MountVolume.SetUp failed for volume "usr-src" : hostPath type check failed: /usr/src is not a directory
```

Reply:
What about [this](https://github.com/DataDog/helm-charts/issues/867#issuecomment-1525224162)?

Reply:
@yellowhat sounds similar :D DataDog gets this option for GKE, can you add a similar option too?

Reply:
I will have a look